Sources/StreamChatUI/ChatChannel/ChatChannelHeaderView.swift

```diff
@@ -5,11 +5,15 @@
 import StreamChat
 import UIKit
 
+// swiftlint:disable colon
+
 /// The view that displays channel information on the message list header
 open class ChatChannelHeaderView:
     _View,
     ThemeProvider,
     ChatChannelControllerDelegate {
+    // swiftlint:enable colon
```

Comment:
I think this rule is actually better if we fix it like this: Otherwise, we will have this disabled rule all over the place. WDYT?
```suggestion
/// The view that displays channel information on the message list header
open class ChatChannelHeaderView: _View,
    ThemeProvider,
    ChatChannelControllerDelegate {
```

Reply:
Yep, great suggestion - I have pushed this change-up :]